dipy/tracking/local/localtrack.pyx

```diff
@@ -143,16 +172,10 @@ def local_tracker(DirectionGetter dg, TissueClassifier tc,
                 Last point is INVALIDPOINT.
 
     """
-    if (seed.shape[0] != 3 or first_step.shape[0] != 3 or
-        voxel_size.shape[0] != 3 or streamline.shape[1] != 3):
-        raise ValueError()
-
     cdef:
-        int i
-        TissueClass tissue_class
+        size_t i
         double point[3], dir[3], vs[3], voxdir[3]
-        double[::1] pview = point, dview = dir
-        void (*step)(double*, double*, double) nogil
+        void (*step)(double * , double*, double) nogil
```

Comment:
The original line was OK, no?

Reply:
Err, yes, sorry, I accidently added a space :) Will remove.